src/js/components/Video/__tests__/Video-test.tsx

```diff
@@ -33,7 +33,7 @@ describe('Video', () => {
         theme={{
           video: {
             controls: { background: '#000000' },
-            scrubber: { track: { color: '#444444' } },
+            scrubber: { track: { color: '#444444' }, interval: 0 },
```

Comment:
I think instead of having to add interval here we should change the interval type definition in `themes/base.d.ts` to be `interval?: number;`